src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -411,6 +411,11 @@ private async Task StartContainerAsync(IExecutionContext executionContext, Conta
                 }
             }
 
+            // Expose docker container name mapping to env
+            var containerEnv = executionContext.Variables.Get(Constants.Variables.Agent.ContainerMapping);
+            containerEnv = (containerEnv != null ? containerEnv + "," : "") + $"{container.ContainerName}:{container.ContainerDisplayName}";
```

Comment:
Shouldn't this be `$"{container.ContainerName}:{container.ContainerId}` rather than ContainerDisplayName?

Reply:
Also, I still feel like this would be better as JSON (for example, `{ containerName: { "id": containerId }, ... }` rather than a idiosyncratic format that can't easily be extended. The task lib could have a function that exposes this to tasks as an object either way, so I don't think we need to optimize for simple parsing from shell scripts.

Reply:
ContainerDisplayName is the value we generate. In the examples onebranch showed us, they are looking for this value (what is passed in as --name when the container is created).

From what I can see, both values work for commands like `docker start <name>`. Do you have a preference on which we should expose?

Reply:
I like JSON better. Let's do it